k8s/airflow/Dockerfile

```diff
@@ -29,9 +29,9 @@ RUN apt-get update && apt-get install -y \
 RUN service cron stop \
     && useradd -ms /bin/bash -d ${AIRFLOW_HOME} airflow
 
-RUN pip3 install --disable-pip-version-check cryptography \
+RUN pip3 install --disable-pip-version-check cryptography==2.4.1 \
     && pip3 install --disable-pip-version-check apache-airflow[postgres,jdbc,password,mysql,celery,redis,s3]==1.9.0 \
-    pyOpenSSL ndg-httpsclient pyasn1 psycopg2-binary redis hiredis requests
+    pyOpenSSL==18.0.0 ndg-httpsclient==0.5.1 pyasn1 psycopg2-binary==2.7.6.1 redis==2.10.6 hiredis==0.2.0 requests
```

Comment:
@aliaksandr-d please pin version of all dependencies. pyasn1 == ?

Reply:
Fixed

Reply:
Fixed